problemsets-mat223/linearalbegra.tex

```diff
@@ -2462,25 +2462,66 @@ \section*{Matrices}
 		\mat{1&-3&0\\0&0&1\\0&0&0}\mat{x\\y\\z}=\vec b.
 	\]
 	\begin{parts}
-		\item If $\vec b=\mat{1\\2\\3}$, is the set of solutions to this system a 
-		point, line, plane, or other?
-		\item If $\vec b=\mat{1\\1\\0}$, is the set of solutions to this system a 
-		point, line, plane, or other?
+		\item If $\vec b=\mat{1\\2\\3}$, is the set of solutions to this system
+			a point, line, plane, or other?
+			\begin{solution}
+				This system has no solutions, since the third equation would be
+				$0=3$, which is impossible. 
+			\end{solution}
+		\item If $\vec b=\mat{1\\1\\0}$, is the set of solutions to this system
+			a point, line, plane, or other?
+			\begin{solution}
+				A line.	The system would be
+				\begin{align*}
+					x - 3y &= 1 \\
+					z &= 1 \\
+					0 &= 0
+				\end{align*}
+				$\vec x=\mat{x\\y\\z}$ satisfies this system if and only if it is
+				on the line $\vec x = t \mat{3\\1\\0} + \mat{1\\0\\1}$.
+			\end{solution}
 	\end{parts}
 
 	\question
-	Let $\vec d_1=\mat{1\\1\\2}$ and
-	$\vec d_2=\mat{-1\\1\\0}$. 
+	Let $\vec d_1=\mat{1\\1\\2}$ and $\vec d_2=\mat{-1\\1\\0}$. 
 	Let $\mathcal P$ be the plane given in vector form by $\vec x=t\vec d_1+s\vec d_2$.
 	Further, suppose $M$ is a matrix so that $M\vec r\in\mathcal P$ for any $\vec r$.
 	\begin{parts}
 		\item How many rows does $M$ have?
+			\begin{solution}
+				Three. It must have three rows in order for $M\vec r$ to be an
+				element of $\R^3$. 
+			\end{solution}
 		\item Find such an $M$.
-		\item Find necessary and sufficient conditions (phrased as equations) for $\vec n$
-			to be a normal vector for $\vec p$.
-		\item Find a matrix $K$ so that solutions
-			to $K\vec x=\vec 0$ are normal vectors for $\mathcal P$.
-			How do $K$ and $M$ relate?
+			\begin{solution}
+				$M=\mat{1&-1&0\\1&1&0\\2&0&0}$ is one possible answer, since if
+				$\vec r = \mat{a\\b\\c}$, then $M \vec r = a \vec d_1 + b \vec d_2$. 
+
+				Another less interesting answer is the zero matrix. 
+			\end{solution}
+		\item Find necessary and sufficient conditions (phrased as equations) for 
+			$\vec n$ to be a normal vector for $\mathcal P$.
+			\begin{solution}
+				$\vec n$ is normal to $\mathcal P$ if and only if 
+				$\vec n\cdot\vec d_1=0$ \emph{and} $\vec n\cdot\vec d_2=0$
+			\end{solution}
+		\item Find a matrix $K$ so that solutions to $K\vec x=\vec 0$ are normal
+			vectors for $\mathcal P$. How do $K$ and $M$ relate?
+			\begin{solution}
+				$K = \mat{1&1&2\\-1&1&0\\0&0&0}$. $K$ and $M$ are transposes of 
+				one another.
+
+				The conditions in the previous part translate to the following 
```

Comment:
Spell it out: the condition Kx=0 translates to...

Reply:
Added the conditions from the previous part explicitly.

The matrix K in this part doesn't need to have its third row in order to get the same system of equations. But I figured you wanted the row of zeroes in there so that this matrix became the transpose of M.